lib/crowbar/client/app/upgrade.rb

```diff
@@ -219,12 +219,12 @@ def backup(component)
           catch_errors(e)
         end
 
-        desc "repocheck ADDON",
-          "Check for existing repositories for an addon (ha|storage)"
+        desc "repocheck COMPONENT",
+          "Check for existing repositories for a component (crowbar|ha|storage)"
```

Comment:
> I understand it calls a method from crowbar module

this herre is completely independent of the model, I have not even thought about it